### PR TITLE
countdownAndlanguagedropdown

### PR DESCRIPTION
**Pull request by GeraldMit**
Contributes to # (add issue number here. i.e. `#302`)
#222
#213

## What did you do?
1) change size of count down timer
image
image

2) add a dropdown for languages
image
image

3) change language selection to native language name for the language
image
image

## Why did you do it?
for issues as given

## How have you tested it?
local build

## Were docs updated if needed?

- [x] No
- [ ] Yes
- [ ] N/A

#### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new console logs
- [x ] Fixes entire issue
- [x] Partial fix for issue

### CODE REVIEW

File: ui/src/components/AppHeader/AppHeader.vue [hunk 1]
```diff
@@ -25,19 +25,53 @@
...
+      <cv-overflow-menu :flip-menu="true" :label="$t('ariaLanguageSetting')">
+        <template slot="trigger">
+          <Language32 />
+        </template>
+        <cv-overflow-menu-item
+          v-for="entry in languages"
+          :key="entry.title"
+          @click="changeLocale(entry.language)"
+        >
+          {{ entry.title }}
```
Comment (GeraldMit) [inline ui/src/components/AppHeader/AppHeader.vue:37]: this had a missing front { in the original push but i fixed it.

**Review by upkarlidder — APPROVED** [review]
LGTM. This is what I found for LTR -> https://medium.com/i18n-and-l10n-resources-for-developers/the-ultimate-guide-to-vue-localization-with-vue-i18n-bf98b1d40c65

Added commit to this PR.

**Comment by davidnixon** [comment]
This branch works great for me locally.
Yesterday I added this patch to the "blue" server to support detecting the browser language.
```diff
--- a/ui/src/plugins/i18n.js
+++ b/ui/src/plugins/i18n.js
@@ -31,7 +31,7 @@ const messages = {
 };
 
 export default new VueI18n({
-  locale: 'en',
+  locale: navigator.language || 'en',
   fallbackLocale: 'en',
   messages
 });
```
What do you think of including that permanently?

File: ui/src/components/AppHeader/AppHeader.vue [hunk 2]
```diff
@@ -25,21 +25,80 @@
...
+        { flag: '', language: 'ja', title: '日本語' },
+        { flag: '', language: 'tl', title: 'Tagalog' },
+        { flag: '', language: 'vn', title: 'ngôn ngữ tiếng Việt' }
+      ]
+    };
+  },
+  methods: {
+    changeLocale(locale) {
+      i18n.locale = locale;
+      this.ltr = !['ar', 'fa'].includes(locale);
```
Comment (davidnixon) [inline ui/src/components/AppHeader/AppHeader.vue:90]: Added this to resolve issue for ar, fa where the language setting menu moved to the left of the screen and then opened to the left which meant to was clipped by the edge of the browser and very difficult to use. With this change , when the language menu is on the left it opens to the right.

File: ui/src/plugins/i18n.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ const messages = {
 };
 
 export default new VueI18n({
-  locale: 'en',
+  locale: navigator.language || 'en',
```
Comment (davidnixon) [inline ui/src/plugins/i18n.js:34]: Set default locale to the browser language

File: ui/src/components/AppHeader/AppHeader.vue (hunk 2 above)
Comment (davidnixon) [reply ui/src/components/AppHeader/AppHeader.vue:90]: ltr menu
image

rtl menu prior to this change. @upkarlidder mentioned this in the chat.
image

**Pull request merged**